service/src/main/java/uk/gov/hmcts/reform/fpl/service/PastHearingDatesValidatorService.java

```diff
@@ -25,6 +25,12 @@
             errors.add("Enter a valid end time");
         }
 
+        if (errors.isEmpty() && hearingStartDate != null && hearingEndDate != null
+            && !hearingEndDate.isAfter(hearingStartDate)
+        ) {
+            errors.add("The end date and time must be after the start date and time");
+        }
```

Comment:
Not one for this ticket but we should refactor this at some point and use javax validation. Should we create a tech debt ticket?

Reply:
Created tech debt ticket: https://tools.hmcts.net/jira/browse/FPLA-2527